sympy/matrices/matrices.py

```diff
@@ -3549,6 +3549,28 @@ def _diagonalize_clear_subproducts(self):
         del self._eigenvects
 
     def jordan_cell(self, eigenval, n):
+        """
+        Generate Jordan Blocks if eigenvalue and dimension is provided.
+
+        https://en.wikipedia.org/wiki/Jordan_matrix
+
+        Examples
+        ========
+        >>> from sympy.matrices import jordan_cell
```

Comment:
You are importing `jordan_cell` from `sympy.matrices.dense.py` rather than the one from `sympy.matrices.matrices.py`